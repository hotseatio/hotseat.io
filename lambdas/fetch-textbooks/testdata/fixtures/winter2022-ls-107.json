[{"crn":"252342200","school":"UCLA","term":"Winter 2022","department":"LIFESCI","courseName":"Genetics","sectionName":"1","title":"Genetics (CRN: 252342200)","items":[{"isbn":"9781260444056","crn":"252342200","longTitle":"GENETICS : FROM GENES TO GENOMES (LOOSE-LEAF)","shortTitle":"GENETICS : FROM GENES TO GENOMES (LOOSE-LEAF)","bookAuthor":"HARTWELL","bookEdition":"7","courseMaterialRequirement":"Required","itemId":"32527","sectionNotes":"","okToSubstitute":false,"itemType":"1","isBook":"T","itemDisplayName":"GENETICS : FROM GENES TO GENOMES (LOOSE-LEAF)","internalid":"4349","storeItem":true},{"isbn":"9781319149581","crn":"252342200","longTitle":"I - CLICKER 2","shortTitle":"I - CLICKER 2","bookAuthor":"HOLTZBRINCK","bookEdition":"","courseMaterialRequirement":"Required","itemId":"33127","sectionNotes":"","okToSubstitute":false,"itemType":"1","isBook":"T","itemDisplayName":"I - CLICKER 2","internalid":"4358","storeItem":true}],"sectionNotes":["","ADVANCED CLICKER CAN BE USED IN PLACE OF CLICKER + SEE 9781498603041 WHEN SO (CLICKER ALONE W/INSTRUC FOR 5YR ICLICKER STUDENT APP)"]}]