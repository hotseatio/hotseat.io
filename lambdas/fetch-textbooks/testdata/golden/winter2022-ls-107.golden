[
  {
    "edition": 7,
    "isRequired": true,
    "isbn": "9781260444056",
    "title": "GENETICS : FROM GENES TO GENOMES (LOOSE-LEAF)",
    "author": "HARTWELL"
  },
  {
    "isRequired": true,
    "isbn": "9781319149581",
    "title": "I - CLICKER 2",
    "author": "HOLTZBRINCK"
  }
]
